### ISSUE TITLE
Adding lots of judgements (FA+/ITG) crashes the game

### ISSUE THREAD

**Issue by inyongthegr8** [dguzek/Simply-Love-SM5#302]
Every time I add more than 25+ assets of judgements from [Zenius-I-Vanisher](https://zenius-i-vanisher.com/v5.2/thread?threadid=8936) to Graphics/_judgements folder, the game crashes while loading the judgements from different modes (FA+/ITG) when holding start button to change your preferred judgement/modify options before the song is selected.

Can reproduce from the prior versions of Simply Love.
Stepmania built from source and 5.0.12.

Not sure if it's an issue from Simply Love itself or StepMania.

**Comment by natano** [comment]
Do you have the same issue with 5.1 or 5.3 (outfox)?

**Comment by inyongthegr8** [comment]
Yes. It happens with [ClubFantastic (5.1)](https://wiki.clubfantastic.dance/) as well.
